### PR TITLE
Sklearn frontend updates

### PR DESCRIPTION
**Pull request by CloudManX**

### CODE REVIEW

File: tests/python/frontend/sklearn/test_forward.py [hunk 1]
```diff
@@ -19,45 +19,53 @@
...
 def _test_model_impl(helper, model, dshape, input_data):
-    helper.compile(model, dshape, "float32")
-    sklearn_out = model.transform(input_data)
+    helper.compile(model, dshape, 'float32', 'transform')
+    if type(model).__name__ == 'ThresholdOneHotEncoder':
+        sklearn_out = model.transform(input_data).toarray()
+    else:
+        sklearn_out = model.transform(input_data)
     tvm_out = helper.run(input_data)
+    print(type(model).__name__)
```
Comment (trevor-m) [inline tests/python/frontend/sklearn/test_forward.py]: Remove debug prints

**Review by trevor-m — CHANGES_REQUESTED** [review]
Thanks @yunma10

**Comment by trevor-m** [comment]
Thanks @yunma10
Please fix the formatting in the CI and we can merge

File: tests/python/frontend/sklearn/test_forward.py (hunk 1 above)
Comment (CloudManX) [reply tests/python/frontend/sklearn/test_forward.py]: Fixed

File: tests/python/frontend/sklearn/test_forward.py [hunk 2]
```diff
@@ -129,24 +135,75 @@ def test_column_transfomer():
...
     test_robust_scaler()
     test_column_transfomer()
     test_threshold_onehot_encoder()
-    test_inverse_label_transformer()
+    # test_robust_ordinal_encoder()
+    test_na_label_encoder()
+    test_standard_scaler()
+    test_kbins_discretizer()
+    # test_tfidf_vectorizer()
+    test_pca()
```
Comment (srinidhigoud) [inline tests/python/frontend/sklearn/test_forward.py]: We dont need this for testing

Comment (CloudManX) [reply tests/python/frontend/sklearn/test_forward.py]: This is the entrance for local testing. This is following the format from other frameworks.

Comment (srinidhigoud) [reply tests/python/frontend/sklearn/test_forward.py]: understood, thanks

**Review by trevor-m — APPROVED** [review]

**Pull request merged**